test/BUILD.gn

```diff
@@ -165,6 +165,7 @@ test("brave_browser_tests") {
     "//brave/browser/ui/webui/brave_new_tab_ui_browsertest.cc",
     "//brave/browser/ui/webui/brave_welcome_ui_browsertest.cc",
     "//brave/common/brave_channel_info_browsertest.cc",
+    "//brave/components/brave_rewards/browser/rewards_service_browsertest.cc",
```

Comment:
Don't add it here, but uncomment it bellow

Reply:
Totally missed that.